### DISCUSSION TITLE
buddhism/2024/workflow/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#74] [General]
# buddhism/2024/workflow/

https://shanenull.com/buddhism/2024/workflow/

**Comment by shane0** [comment]
Duality is where we create a false division in phenomena based on two extremes, such as good or bad, true or false, pure or impure, etc.

**Comment by shane0** [comment]
# topics

**Reply by shane0** [reply]
# eat 

https://shanenull.com/buddhism/2023/eat/
